src/dmd/parse.d

```diff
@@ -3684,6 +3684,13 @@ final class Parser(AST) : Lexer
             check(TOK.rightParentheses);
             break;
 
+        case TOK.else_:
```

Comment:
If this is added instead as:
```
if (token.value == TOK.else_)
    errorSupplemental(token.loc, ...)
```
then this change adds only two lines of code instead of 6.

Reply:
Added under the `default:` section, that is.

Reply:
oh thanks, makes sense now that the hint is a separate message